build/docker/agent/Dockerfile

```
@@ -102,6 +102,12 @@
 
 ENV CARLA_ROS_BRIDGE_ROOT=/catkin_ws/src/ros-bridge
 
+# Install teb_local_planner as well as teb_planner_pa
+RUN sudo apt-get install -y \
```

Hadolint (2.12.0)

[error] Do not use sudo as it leads to unpredictable behavior. Use a tool like gosu to enforce root

(DL3004)